artifact.go

```diff
@@ -29,38 +29,7 @@ import (
 	"golang.org/x/exp/slices"
 	"golang.org/x/sync/semaphore"
 
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/buildinfo"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/executable"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/imgconf/apk"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/c/conan"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/dotnet/deps"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/dotnet/nuget"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/golang/binary"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/golang/mod"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/java/gradle"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/java/jar"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/java/pom"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/nodejs/npm"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/nodejs/pkg"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/nodejs/pnpm"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/nodejs/yarn"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/php/composer"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/python/packaging"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/python/pip"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/python/pipenv"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/python/poetry"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/ruby/bundler"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/ruby/gemspec"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/rust/binary"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/rust/cargo"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/licensing"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/os/alpine"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/os/amazonlinux"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/os/debian"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/os/mariner"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/os/redhatbase"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/os/release"
-	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/os/ubuntu"
+	_ "github.com/aquasecurity/trivy/pkg/fanal/analyzer/all"
```

Comment:
Why not keep it as before? Due to this import we now need to deregister analyzer